src/lib/child_processes/child_processes.ml

```diff
@@ -162,9 +162,10 @@ type output_handling =
   * [`Pipe | `No_pipe]
   * [`Keep_empty | `Filter_empty]
 
-(** Given a Reader.t coming from a process output, optionally log the lines
-    coming from it and return a strict pipe that will get the lines if the
-    argument is `Pipe and be empty if it's `No_pipe. *)
+(* Given a Reader.t coming from a process output, optionally log the lines
```

Comment:
Should this be a doccoment? E.g. `(** xxx *)`.

Reply:
I changed it from a doccomment, because this function is not exposed via the .mli file, so not usable from outside this module.